src/js/pensions/config/form.js

```diff
@@ -96,6 +96,11 @@ function isBetween18And23(childDOB) {
   return moment(childDOB).isBetween(moment().startOf('day').subtract(23, 'years'), moment().startOf('day').subtract(18, 'years'));
 }
 
+// Actually checks if they're under 17.75, but that doesn't make for a good function name
+function isUnder18(childDOB) {
```

Comment:
Could maybe go with a non-numerical descriptor such as 'isEligibleForDisabilitySupport' and then provide comments explaining how this is defined?